### DISCUSSION TITLE
Webmin o alternativa dietro Reverse Proxy?

### DISCUSSION THREAD

**Discussion by labattazza** [indomus/forum#1713] [Networking]
ciao, non so se nel forum c'è qualcuno che è riuscito a trovare il modo di far funzionare Webmin o un software equivalente dietro reverse proxy.

Nelle ricerche che ho fatto mi pare di aver capito che è un compito non facile, qualcuno è riuscito a far funzionare webmin ma con soluzioni non definitive, e solo usando Ngix.

Io nella configurazione di casa utilizzo Caddy come reverse proxy, e con questo non ho trovato nessuno che abbia avuto successo.

Avete qualche suggerimento? Mi farebbe comodo accedere dall'esterno al server per qualche controllo/configurazione, senza esporre altre porte.

Grazie!

**Comment by snakuzzo** [comment]
In realtà puoi farlo semplicemente proxandolo così come avrai proxato Home Assistant.

Se invece ti riferisci al voler visualizzare webmin in http su dashboards di HA (che è in https)... devi esporlo PER FORZA in https con un suo FQDN. Altrimenti ti sviluppi una sorta di `ingress` così com'è presente su HassOS.

**Reply by labattazza** [reply]
ciao, effettivamente riesco a raggiungere webmin come faccio con Home assistant o Nextcloud.
I problemi si presentano una volta effetutato il login, a quel che ho capito webmin non reindirizza nella maniera corretta.
E capita anche disabilitando l'SSL nativo.

Riprovo in serata, ma ho seri dubbi di aver successo stavolta.
Intanto grazie della risposta!
